### PR TITLE
FVM & Metamask tutorials

### PR DESCRIPTION
**Pull request by enoldev**
This PR includes:
- FVM intro tutorial
- Metamask introduction
- Metamask tutorials (create wallet and switch network)
- Feature to enable "Before You Begin" section with recommended tutorials

### CODE REVIEW

File: content/en/tutorials/metamask-intro/what-is-metamask/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+draft: false
+menu:
+    tutorials:
+        parent: "tutorials-metamask-intro"
+weight: 1
+category: lecture
+---
+
+**Metamask** is a crypto wallet and a gateway to blockchain apps.
+The main feature of Metamask is its browser extension, which is available for Brave, Chrome, Edge, Firefox, and Opera.
```
Comment (walkerlj0) [inline content/en/tutorials/metamask-intro/what-is-metamask/index.md:13]:
```suggestion
**Metamask** is a crypto wallet and a gateway to blockchain apps. A crypto wallet is an address on the blockchain the points to an account that can be used to store and exchange cryptocurrency. There are different things you can use to manage your wallet, and Metamask is an application that makes it easy for applications in your browser to send and receive funds, NFTs, and more to the wallet.
The main feature of Metamask is its browser extension, which is available for Brave, Chrome, Edge, Firefox, and Opera.
```

**Comment by walkerlj0** [comment]
Only other comment @enoldev is to make this in dark mode: 
Screen Shot 2023-01-23 at 10 30 58 AM

**Review by walkerlj0 — APPROVED** [review]

File: content/en/tutorials/fvm/deploying-first-contract/index.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+1. Back in Remix, open the **Deployed Contracts** dropdown, within the **Deploy** sidebar tab.
+
+   ![Remix showing my deployed contracts.](remix-deployed-contracts-dropdown.png)
+
+1. Expand the `mint` method. You must fill in two fields here: `to` and `amount`.
+
+   ![Remix showing the mint method of a contract.](remix-expand-mint-method.png)
+
+1. The `to` field specifies where address you want these initial tokens sent to. Open MetaMask, copy your address, and paste it into this field.
```
Comment (mrodriguez3313) [inline content/en/tutorials/fvm/deploying-first-contract/index.md:144]: Is this what was meant?
```suggestion
1. The `to` field specifies what address you want these initial tokens sent to. Open MetaMask, copy your address, and paste it into this field.
```
or 
```suggestion
1. The `to` field specifies where you want these initial tokens sent to. Open MetaMask, copy your address, and paste it into this field.
```

**Review by mrodriguez3313 — COMMENTED** [review]
With respect to the "Deploying a Smart Contract" Tutorial, I found it strange that the tutorials were linked at the top and that and their content was in the pages too. I personally don't like the idea of duplicating that content. thoughts?
Screen Shot 2023-01-24 at 9 39 05 AM

This was one funny thing I saw with the page in dark mode
Screen Shot 2023-01-24 at 9 15 28 AM

**Comment by mrodriguez3313** [comment]
@enoldev nvm 😄 Just pulled in the changes!

**Pull request merged**